### PR TITLE
started phenotyping queries

### PR DESCRIPTION
**Pull request by Samad198**
Output with 8507 hardcoded as the concept id is 
image

### CODE REVIEW

File: api/mapping/management/commands/create_concept.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        #print(scan_report_concepts.values())
+        for scan_report_concept in scan_report_concepts:
+            if (scan_report_concept.content_type_id==17):
+                # If it is for a value
+                scan_report_values = ScanReportValue.objects.filter(pk=scan_report_concept.object_id)
+                scan_value = scan_report_values[0]
+                scan_report_fields = ScanReportField.objects.filter(pk=scan_value.scan_report_field_id)
+                scan_field = scan_report_fields[0]
+                finalList.append((scan_field.name ,scan_value.value,scan_value.value_description))
+            if (scan_report_concept.content_type_id==15):
```
Comment (spco) [inline api/mapping/management/commands/create_concept.py:30]: use `elif` for clarity

File: api/mapping/management/commands/create_concept.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+)
+from django.http import HttpRequest
+
+
+class Command(BaseCommand):
+    help = 'create rules from existing concepts found for a given table id'
+
+    def handle(self, *args, **options):
+
+        concept_id = 8507 # id for male
```
Comment (spco) [inline api/mapping/management/commands/create_concept.py:17]: please set this as an input argument rather than hardcoded

**Review by spco — COMMENTED** [review]
Great stuff in general, a few comments to tidy it up.

The remaining piece of logic I'd like (sorry, hadn't mentioned this before now!) is that we should have an optional argument to indicate whether to use all scan reports, or to ignore mappings from archived reports. Could you add this please? Call it `use_archived`, defaulting to `False`, and if `False` then perform a check or filter to only extract mappings associated to fields or values in ScanReports where the column `hidden=False`.

If you could also add a few comments to really make the logic crystal clear that would be useful.

File: api/mapping/management/commands/create_concept.py (hunk 1 above)
Comment (Samad198) [reply api/mapping/management/commands/create_concept.py]: Done

File: api/mapping/management/commands/create_concept.py (hunk 2 above)
Comment (Samad198) [reply api/mapping/management/commands/create_concept.py]: Done

**Comment by Samad198** [comment]
> Great stuff in general, a few comments to tidy it up.
> 
> The remaining piece of logic I'd like (sorry, hadn't mentioned this before now!) is that we should have an optional argument to indicate whether to use all scan reports, or to ignore mappings from archived reports. Could you add this please? Call it `use_archived`, defaulting to `False`, and if `False` then perform a check or filter to only extract mappings associated to fields or values in ScanReports where the column `hidden=False`.
> 
> If you could also add a few comments to really make the logic crystal clear that would be useful.

Done

**Pull request closed without merging**